.github/workflows/publish-java-cdk-command.yml

```diff
@@ -101,7 +108,8 @@ jobs:
           arguments: --scan :airbyte-cdk:java:airbyte-cdk:cdkBuild
 
       - name: Check for Existing Version
-        if: ${{ !(env.FORCE == 'true') }}
+        # we only check existing version if it's a manual trigger and FORCE is set to false
+        if: ${{ (env.FORCE != 'true') }}
```

Comment:
I never was a fan of `!(... == ...)`.

Reply:
I think they're actually different...
`!(env.FORCE == 'true')` returns `false` only if `FORCE==true` while `(env.FORCE != 'true')` returns `false` all the time, except if `FORCE == true`